### DISCUSSION TITLE
C2668	'ImGui::SliderFloat': ambiguous call to overloaded function

### DISCUSSION THREAD

**Discussion by whitechopchicken** [ocornut/imgui#6197] [New Users Build/Link/Run issues ONLY!]
When I built it, I reported it wrong
![error](https://user-images.githubusercontent.com/88885300/221370764-f7aa5d28-8243-4c9f-9657-fe9c50a5299e.png)

**Comment by emoon** [comment]
The SliderFloat function looks like this
```C++
SliderFloat(const char* label, float* v, float v_min, float v_max, const char* format = "%.3f", ImGuiSliderFlags flags = 0);
```
So you shouldn't pass that `3.` at the end

**Comment by whitechopchicken** [comment]
I commented out the "3." and it compiled. But I'm not sure how this will affect the original program. Thanks for your reply.




...
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "ocornut/imgui"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年2月26日(星期天) 凌晨2:29
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [ocornut/imgui] C2668 'ImGui::SliderFloat': ambiguous call to overloaded function (Discussion #6197)





 
The SliderFloat function looks like this
 SliderFloat(const char* label, float* v, float v_min, float v_max, const char* format = "%.3f", ImGuiSliderFlags flags = 0);
 
So you shouldn't pass that 3. at the end
